app/services/timer-service.js

```diff
@@ -0,0 +1,43 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+  _countdown() {
+    if (this.get('seconds') === 0) {
+      this.set('timeUp', true);
+    } else {
+      this.incrementProperty('seconds', -1);
+      this._timer = Ember.run.later(this, '_countdown', 1000);
+    }
+  },
+  elapsedTime() {
+    return this.get('timeLimit') - this.get('seconds');
+  },
+  iconState: 'glyphicon-pause',
+  timeUp: false,
+  pause() {
+    Ember.run.cancel(this.get('_timer'));
+    this.set('_timer', null);
+    this.set('iconState', 'glyphicon-play');
```

Comment:
I'd like to see our icons in their own hash, so we can give them internal names `play` `pause` etc - to decouple this from glyphicon stuff.

```
{'play': 'glyphicon-play}
icons['play']
```